### PR TITLE
(feat)KHP3-7086-Add Paid bill receipt print preview

### PR DESCRIPTION
**Pull request by Omoshlawi**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

https://github.com/user-attachments/assets/43be67fb-d291-4964-8459-d9e9b5877aa2

## Related Issue

https://thepalladiumgroup.atlassian.net/browse/KHP3-7086

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/invoice/print-bill-receipt/print-bill-receipt.resource.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const fetcher = async (url: string) => {
+  const res = await fetch(url, {
+    headers: {
+      'Content-Type': 'application/pdf',
+    },
+  });
+  const fileData = await res.arrayBuffer();
+  const blob = new Blob([fileData], { type: 'application/pdf' });
+  const _url = URL.createObjectURL(blob);
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/invoice/print-bill-receipt/print-bill-receipt.resource.ts:12]:
```suggestion
  return URL.createObjectURL(blob);
```

File: packages/esm-billing-app/src/invoice/print-bill-receipt/receipt-print-preview.modal.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        {error && <ErrorState error={error} headerTitle={t('previewError', 'Preview Error')} />}
+        {url && !isLoading && <iframe src={String(url)} title="Receipt Preview" className={styles.previewFrame} />}
+      </ModalBody>
+      <ModalFooter>
+        <Button kind="secondary" onClick={onClose} type="button" className={styles.btn}>
+          {t('close', 'Close')}
+        </Button>
+        {/* <Button type="submit" disabled={isLoading || error} onClick={() => window.open(url as any, '_blank')}>
+          {t('print', 'Print')}
+        </Button> */}
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/invoice/print-bill-receipt/receipt-print-preview.modal.tsx:39]:
```suggestion
     
```

**Review by donaldkibet — DISMISSED** [review]
Thanks @Omoshlawi LGTM 🚀

File: packages/esm-billing-app/src/invoice/print-bill-receipt/receipt-print-preview.modal.tsx (hunk 2 above)
Comment (Omoshlawi) [reply packages/esm-billing-app/src/invoice/print-bill-receipt/receipt-print-preview.modal.tsx]: @donaldkibet fixed as per the suggestions

**Review by donaldkibet — APPROVED** [review]

**Comment by donaldkibet** [comment]
From the video, printing is only possible by clicking the print icon within the PDF preview. To improve user experience, we should add the ability to trigger the browser's native print dialog directly from the modal actions. This would provide users with an additional, more familiar printing option. cc @ojwanganto @Omoshlawi

**Pull request merged**